applications/sandbox/systems/explosion_particlesystem.hpp

```diff
@@ -11,14 +11,14 @@ using namespace legion;
 struct explosionParameters
 {
     math::vec3 startingSize;
-    rendering::material_handle particleMaterial;
-    rendering::model_handle particleModel;
+    material_handle particleMaterial;
+    model_handle particleModel;
     float maxLifeTime;
```

Comment:
using namespace legion doesn't just give you all the sub-namespaces as well. this shouldn't compile